File: study/22.10.25/EJH/문자열 다루기 기본.py
def solution(s):
    answer = True
    num = set()
    for i in range(0, 10):
        num.add(str(i))
    if len(s) == 4 or len(s) == 6:
        for word in s:
            if word not in num:
                answer = False
                break
    else:
        answer = False
    return answer

print(solution("a234"))
print(solution("1234"))

File: study/2022/22.10.25/PKB/문자열 다루기 기본.java
class Solution {

    public boolean solution(String s) {

        int l = s.length();

        if (l != 4 && l != 6) return false;

        for (int i = 0; i < l; i++) {
            char c = s.charAt(i);

            if (c < '0' || c > '9') return false;
        }

        return true;

    }

}
